how-to/security/user-management.md

```diff
@@ -9,6 +9,34 @@ Ubuntu developers decided to disable the administrative root account by default
 
 Instead, the `sudo` utility ("superuser do") is used to carry out system administrative duties. `sudo` allows an authorised user to temporarily elevate their privileges using their own password instead of having to know the password belonging to the root account. This provides accountability for all user actions, and gives the administrator control over which actions a user can perform with said privileges.
 
+### sudo-rs
+
+Since Ubuntu 25.10 (Questing Quokka), `sudo` is provided by the `sudo-rs` package (a rust implementation). While `sudo-rs` is not a perfect in-place replacement for sudo.ws, most common use cases are supported and the change should be invisible to most users.
```

Comment:
```suggestion
Since Ubuntu 25.10 (Questing Quokka), `sudo` is provided by the `sudo-rs` package (a rust implementation). While `sudo-rs` is not 100% compatible with 
sudo.ws, the majority of common use cases are supported and the change should be invisible to most users.
```